scss/_rc-custom-adjustments.scss

```diff
@@ -0,0 +1,180 @@
+// Real Crowd Custom Bootsrap overrides
+//
+// Copy class names from specific files ex. `_breadcrumb.scss` to this file to override default conventions (ex. link colors) or add extra UI details (ex. underline on active link states) without modifying source files.
+
+
+// Table of Contents
+
+// Variables
+
+// Sitewide Typography
+// Breadcrumbs
+// Buttons
+// Cards
+// Dropdowns
+// Forms
+// Navigation
+// Pagination
+
+
+//Variables
+
+$box-shadow: 0 0 13px 0 rgba($black,.15);
+
+
+
+// Sitewide Typography
+
+body {
+  // Improves sitewide typogrpahy legibility by using built in kerning (letterspacing math, ex spacing between Wa combination) of the typeface, resulting in more readable and overall better looking text.
+  text-rendering: optimizeLegibility;
+
+  // Activates improved rendering of typography on webkit based browsers, making text closer to its intended look, as it was designed and appears in print.
+  -webkit-font-smoothing: antialiased;
```

Comment:
again, keep cross-browser in mind with these advanced values.

https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth